### PR TITLE
Update Supported Versions doc

### PR DESCRIPTION
**Pull request by ericvn**
Please provide a description for what this PR is for.

Replacement for https://github.com/istio/istio.io/pull/12249 which is needing a lint fix.

### CODE REVIEW

File: content/en/docs/releases/supported-releases/index.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ Please keep up-to-date and use a supported version.
 
 | Minor Releases   | Patched versions with no known CVEs           |
 |------------------|-----------------------------------------------|
+| 1.16.x           | 1.16.0+                                       |
 | 1.15.x           | 1.15.3+                                       |
 | 1.14.x           | 1.14.5+                                       |
-| 1.13.x           | 1.13.9+                                       |
-| 1.12 and earlier | None, all versions have known vulnerabilities |
+| 1.13 and earlier | None, all versions have known vulnerabilities |
```
Comment (ericvn) [inline content/en/docs/releases/supported-releases/index.md:85]: Technically 1.13.9 doesn't have a known vulnerability, at least I don't see where we had a fix shown in https://istio.io/latest/news/ which called out 1.13.9. But 1.13 is EOL. We could call out releases with no CVEs that have reached EOL, 1.14 will be there soon, with some different text, such as:

```suggestion
| 1.13                    | End of life. A new CVE will NOT be patched.  |
| 1.12 and earlier | None, all versions have known vulnerabilities |
```

File: content/en/docs/releases/supported-releases/index.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ Please keep up-to-date and use a supported version.
 
 | Minor Releases   | Patched versions with no known CVEs           |
 |------------------|-----------------------------------------------|
+| 1.16.x           | 1.16.0+                                       |
```
Comment (kfaseela) [inline content/en/docs/releases/supported-releases/index.md:82]: we already released 1.16.1 , so should this be 1.16.1+ ?

Comment (ericvn) [reply content/en/docs/releases/supported-releases/index.md:82]: 1.16.0 doesn't have any CVE's so it's still 1.16.0. We only increase this value when we have an actual CVE. Example: 1.15.3 was determined as https://istio.io/latest/news/security/istio-security-2022-008/ says 1.15.2 has a CVE. 1.14 was determined by https://istio.io/latest/news/security/istio-security-2022-007/.

File: content/en/docs/releases/supported-releases/index.md (hunk 1 above)
Comment (ericvn) [reply content/en/docs/releases/supported-releases/index.md:85]: Any feedback on if the second suggestion is better or worse would be helpful. I can make the change if needed.

Comment (kfaseela) [reply content/en/docs/releases/supported-releases/index.md:85]: Second suggestion looks better to me.

Comment (ericvn) [reply content/en/docs/releases/supported-releases/index.md]: Updated adding a separate 1.13 line.

**Review by kfaseela — APPROVED** [review]

**Pull request merged**